### ISSUE TITLE
"The property GSettings:schema is deprecated"

### ISSUE THREAD

**Issue by mgedmin** [gtimelog/gtimelog#51]
On Ubuntu 14.04 I get this warning:

```
$ git checkout experimental-ui && ./mockup.py --version
GTimeLog version: 0.10.dev0
Python version: 3.4.0 (default, Jun 19 2015, 14:20:21) [GCC 4.8.2]
GTK+ version: 3.10.8
Data directory: /home/vagrant/.local/share/gtimelog
./mockup.py:147: Warning: The property GSettings:schema is deprecated and shouldn't be used anymore. It will be removed in a future version.
  gsettings = Gio.Settings.new("org.gtimelog")
Legacy config directory: /home/vagrant/.config/gtimelog
Settings will be migrated to gsettings (org.gtimelog) on first launch
```

PyGI version is 3.12.0 (differs from the GTK+ version, curiously enough).

**Comment by mgedmin** [comment]
Seems relevant: https://bugzilla.gnome.org/show_bug.cgi?id=732102

**Comment by mgedmin** [comment]
"Fixed" as part of #57, specifically by 33a14e4735643be7cf59a86146805a2985523183.
